dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/webcontainerext/JspDependent.java

```diff
@@ -93,7 +93,12 @@ public boolean isOutdated() {
     			}
     			// end 213703: add logging for isoutdated checks
             }
-        }                
+        } else {
```

Comment:
Since this work was done in 2022 it looks the commit is only 6 days old lets update the copyright date range here to include 2022